.bazelrc

```diff
@@ -13,6 +13,10 @@ build:no-tty --curses no
 build:no-tty --progress_report_interval 10
 build:no-tty --show_progress_rate_limit 10
 
+# Disable gflags and glog for building shared libs.
```

Comment:
Same as our CMake build. The main reason is that they contain some static gobals, which causes `double free or corruption` when they're statically linked into multiple shared libraries.